include/fiction/io/write_operational_domain.hpp

```diff
@@ -25,11 +26,17 @@ struct write_operational_domain_params
     /**
      * The tag used to represent the operational value of a parameter set.
      */
-    std::string_view operational_tag = "operational";
+    std::string_view operational_tag = "1";
     /**
      * The tag used to represent the non-operational value of a parameter set.
      */
-    std::string_view non_operational_tag = "non-operational";
+    std::string_view non_operational_tag = "0";
+    /**
+     * Whether to write non-operational samples to the CSV file. If set to `false`, only operational samples are
+     * written. This yields a significantly smaller CSV file. It is recommended to set this option for 3D plots because
+     * the non-operational samples would shadow the operational samples anyway.
+     */
+    bool write_non_operational_samples = true;
```

Comment:
@marcelwa If you like, we could also use an  ``enum``.

Reply:
I absolutely do :smile: